### DISCUSSION TITLE
new behavior for the -time parameter

### DISCUSSION THREAD

**Discussion by Hoover889** [Serene-Arc/bulk-downloader-for-reddit#589] [Ideas]
so as far as I understand the -time parameter just sets the filter that you get when showing top posts in any subreddit ("...SubRedditName/top/?sort=top&t=year")

would it be possible to change the behavior (or add it as a new parameter to do this) so that you can specify any arbitrary date/time  and it will stop downloading once that date is reached (when sorting by new)

Currently if I try to maintain an archive of a reddit the program wastes time by searching through many posts that have already been downloaded when all I really wanted was to download new posts since the last time the script was run.

**Comment by Serene-Arc** [comment]
There is already a way to do what you want, using the `--exclude-id-file` option and the scripts contained in [here](https://github.com/aliparlakci/bulk-downloader-for-reddit/tree/master/scripts).

Introducing a backwards-incompatible change would require releasing a new major version, something that seems bad practice for something so small. The `--time` parameter is the same as Reddit's time option, so it shouldn't be changed if possible.
